tensorflow_addons/options.py

```diff
@@ -37,13 +36,38 @@
 import ...
 import ...
 
-tfa.options.TF_ADDONS_PY_OPS = True
+tfa.options.disable_custom_kernel()
 ```
 """
 
 
 def warn_fallback(op_name):
     warning_msg = FALLBACK_WARNING_TEMPLATE.format(traceback.format_exc(), op_name)
     warnings.warn(warning_msg, RuntimeWarning)
-    global TF_ADDONS_PY_OPS
-    TF_ADDONS_PY_OPS = True
+    global _TF_ADDONS_PY_OPS
+    _TF_ADDONS_PY_OPS = True
+
+
+def enable_custom_kernel():
+    """Prefer custom kernel over pure python kernel.
+
+    Enable using custom CUDA/C++ Kernel instead of pure python kernel. Use this instead of
+    directly accessing the global variable."""
```

Comment:
```
    """Prefer custom kernel to pure python kernel.

    Enable using custom CUDA/C++ kernel instead of pure python kernel.
    It has the same effect as setting environment variable `TF_ADDONS_PY_OPS=0`.
    """
```